english/check-phone/open-message/open-message.md

```diff
@@ -3,3 +3,10 @@ white. Two black dots appear, slowly morphing into eyes as the white of
 the screen takes over your whole field of vision. 
 
 The marshmallows are here.
+What do you do?
+
+[Become a marshmallow.](https://github.com/mjjellybean02/create-your-own-adventure/blob/master/english/becomemarshmallow/becomemarshmallow.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.